AndelaOSP/art-dashboard
# Ft admin can see all users 158914606

## What does this PR do?
Enables admin to view users

## Description of Task to be completed?
Scenario: viewing users
Given A logged in admin
When I navigate to the `Users` page
Then i should see all users in the system

## How should this be manually tested?
On login, click on the Users menu item on the sidebar. This should display the users table.

## What are the relevant pivotal tracker stories?
[158914606](https://www.pivotaltracker.com/story/show/158914606)

## Any background context you want to add?
N/A

## Important notes
N/A

## Packages installed
N/A

## Deployment note
N/A 

## Related PRs branch | PR (branch_name) | (pr_link)
Branch | PR
------ | ------
other_pr_production | [link]()
other_pr_master | [link]()
## Todos
## Screenshots
<img width="1189" alt="screen shot 2018-07-13 at 17 59 22" src="https://user-images.githubusercontent.com/27014080/42698419-830eb450-86c6-11e8-8d2f-0f2308b9ccea.png">

<img width="1343" alt="screen shot 2018-07-16 at 2 01 09 pm" src="https://user-images.githubusercontent.com/26274319/42760013-d0fc38f4-8900-11e8-817d-f252aa400090.png">
